packages/expo/package.json

```diff
@@ -21,8 +21,11 @@
     "dist"
   ],
   "scripts": {
-    "build": "tsc -p tsconfig.build.json",
-    "dev": "tsc -p tsconfig.build.json --watch",
+    "build": "tsup",
+    "dev": "tsup --watch",
+    "dev:publish": "npm run dev -- --env.publish",
+    "build:declarations": "tsc -p tsconfig.declarations.json",
```

Comment:
I know that we do this in the other packages but IMO we should just use `tsup` to also output the types and not use TSC

But that can happen in another PR